LaunchAPPL/Client/MiniVMac.cc

```diff
@@ -370,7 +423,7 @@ bool MiniVMac::CheckOptions(variables_map &options)
         && options.count("minivmac-dir") != 0
         && options.count("minivmac-rom") != 0
         && options.count("system-image") != 0
-        && options.count("autoquit-image") != 0;
+        && (options.count("autoquit-image") + options.count("autquit7-image")) == 1;
```

Comment:
This might lead to a confusing message that the minivmac launch method "needs more configuration" if both are specified. So maybe we should use `>= 1` and abort with a specific error message later?

Or we could get fancy, allow both to be specified, and then examine some 'vers' resource in the System file to pick the right one. This might allow for more convenient switching between system versions.